docs/cli.md

```diff
@@ -71,21 +71,20 @@ For example,
 
     $ docker-compose run web python manage.py shell
 
-will start the `web` service and then run `manage.py shell` in python.
-Note that by default, linked services will also be started, unless they are
-already running.
+starts the `web` service and then runs `manage.py shell` in python. Note that
+by default, linked services are also started, unless they are already running.
 
 One-off commands are started in new containers with the same configuration as a
-normal container for that service, so volumes, links, etc will all be created as
+normal container for that service, so volumes, links, etc are all created as
 expected. When using `run`, there are two differences from bringing up a
 container normally:
 
-1. the command will be overridden with the one specified. So, if you run
-`docker-compose run web bash`, the container's web command (which could default
-to, e.g., `python app.py`) will be overridden to `bash`
+1. The command is overridden with the one specified. So, if you run
```

Comment:
These should just be bullets.  You should only use a numbered list on procedures or steps in a process.